.changeset/gold-pets-hope.md

```diff
@@ -0,0 +1,7 @@
+---
+'@leafygreen-ui/card': patch
+'@leafygreen-ui/tokens': patch
+'@leafygreen-ui/tooltip': patch
+---
+
+Make Card box shadow darker
```

Comment:
Why does this impact tokens and tooltip?

Reply:
whoops, picked the packages that changeset suggested too quickly